### PR TITLE
Documentation for Configuring Sphinx

### PR DESCRIPTION
**Pull request by mohit**
in progress: https://github.com/Clever/sphinx/issues/32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,122 @@ Response headers:
     X-RateLimit-Reset: 1394506274
     X-RateLimit-Bucket: authorized-users
 
+## [Configuring Sphinx](./example.yaml)
+
+Rate limiting in Sphinx is managed by setting up `limits` in `yaml` configuration file.
```
Comment (azylman) [inline README.md]: "in" -> "in a"

File: README.md [hunk 2]
```diff
@@ -74,6 +74,122 @@ Response headers:
...
     X-RateLimit-Bucket: authorized-users
 
+## [Configuring Sphinx](./example.yaml)
+
+Rate limiting in Sphinx is managed by setting up `limits` in `yaml` configuration file. 
+Details about the configuration format can be found in the `[annotated example](./example.yaml)`
+
+It is important to understand the concept of `buckets` and `limits` to effectively configure a rate limiter.
+
+_Limit_: Limits define rate limiting policies that Sphinx enforces by counting requests in named _buckets_
```
Comment (azylman) [inline README.md]: For consistency with bucket, make it singular: "A limit defines a rate limiting policy that Sphinx enforces by counting requests in named _buckets_."

File: README.md [hunk 3]
```diff
@@ -74,6 +74,122 @@ Response headers:
...
+_Bucket_ A bucket is simply a named value. Each request that matches a limit increments the value of one bucket.
```
Comment (azylman) [inline README.md]: missing colon

File: README.md [hunk 4]
```diff
@@ -74,6 +74,122 @@ Response headers:
...
+
+Below is an example of a limit and three requests that increment two bucket values.
+
+### Test Limit
```
Comment (azylman) [inline README.md]: Can we show the config that generates this limit?

Comment (azylman) [inline README.md]: It looks like we do below, but I might like to put it here. Something like "If you want this limit, here's how you would do it, and here's how these three requests would behave"

**Comment by dlsniper** [comment]
Hi. I'd like to ask if it's possible to merge this and then have a different PR / person fix the reminder of the feedback rather than having this waiting here. What do you think? Thank you.

**Comment by mohit** [comment]
thanks @dlsniper this slipped through the cracks. Updated the Readme with feedback from Alex and merging this.

**Pull request merged**